src/pages/candidate-details/candidate-details.ts

```diff
@@ -4,23 +4,28 @@ import { IonicPage, NavController, NavParams, Platform } from 'ionic-angular';
 import { Observable } from 'rxjs/Observable';
 import { Subscription } from 'rxjs/Subscription';
 import { map } from 'rxjs/operators';
+import { zip } from 'rxjs/observable/zip';
 import { merge } from 'rxjs/observable/merge';
-
 import { BasePageComponent } from '../../classes/base-page';
 import { AuthenticationProvider } from '../../providers/authentication/authentication';
 import { StoreModel } from '../../common/store.model';
 import { TestCategory, testCategoryIcons } from '../../common/test-category';
 import { getJournalState } from '../journal/journal.reducer';
-import { getSlotById, getSlots, getCandidateName, getTime, getDetails } from './candidate-details.selector';
 import { Details } from './candidate-details.model';
 import { ClearChangedSlot } from '../journal/journal.actions';
 import { AnalyticsProvider } from '../../providers/analytics/analytics';
 import {
-  AnalyticsScreenNames
+  AnalyticsScreenNames, AnalyticsEventCategories, AnalyticsEvents, AnalyticsDimensionIndices
```

Comment:
Can we go either single line or one-per-line on imports? The linter should have moaned about this, needs looking into.

Reply:
It would be good to decide one way or the other - import { IonicPAge, NavController, NavParams, Platform }  already existed.  So do I convert them to multi line as well?

Reply:
tidied up.